code/controllers/master.dm

```diff
@@ -199,11 +199,14 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 	// Sort subsystems by display setting for easy access.
 	sortTim(subsystems, /proc/cmp_subsystem_display)
 	// Set world options.
-	if(sleep_offline_after_initializations)
-		world.sleep_offline = TRUE
 	world.fps = CONFIG_GET(number/fps)
 	var/initialized_tod = REALTIMEOFDAY
+
+	world.TgsInitializationComplete()
+	if(sleep_offline_after_initializations)
+		world.sleep_offline = TRUE
 	sleep(1)
```

Comment:
I put it here, don't know if you want it anywhere else. It's unimplemented in v3 but, like you suggest, the plan is to start the world in the background and then transfer the players to it when the current round ends. If you want it somewhere else speak now or forever hold your peace.